src/jenkins.py

```diff
@@ -610,7 +597,34 @@ def get_updatable_version(proxy: state.ProxyConfig | None = None) -> str | None:
     return latest_version
 
 
-def download_stable_war(container: ops.Container, version: str) -> None:
+def has_lts_updates(proxy: state.ProxyConfig | None = None) -> bool:
```

Comment:
I think this function is intended to return whether the latest LTS version has a new patch version that can be installed, right? It does currently read like it is looking for whether there is a new LTS version

Reply:
Yes, I think it might make more sense within the context of the charm's `_update_jenkins_version` function, where the flow goes from `has_lts_updates` -> `update_jenkins` :), thank you for the comment!

Reply:
maybe `has_updates_for_lts`? That way it indicates that the check is whether there is an update for the lts version?

Reply:
Updated the function name, thank you for the suggestion!